### PR TITLE
Modified UI user list

### PR DESCRIPTION
**Pull request by jamestsay1207**

### CODE REVIEW

File: components/UserList/index.js [hunk 1]
```diff
@@ -1,21 +1,113 @@
...
 import PropTypes from 'prop-types';
-import Link from 'next/link';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+// add MailIcon
+import MailIcon from '@material-ui/icons/Mail';
```
Comment (jserv) [inline components/UserList/index.js]: Mail icon is a bit misleading. Can you clarify?

**Comment by jserv** [comment]
Advise about writing bette Git commit messages: [How to Write a Git Commit Message](https://blog.louie.lu/2017/03/21/%E5%A6%82%E4%BD%95%E5%AF%AB%E4%B8%80%E5%80%8B-git-commit-message/)

File: components/UserList/index.js [hunk 2]
```diff
@@ -1,21 +1,113 @@
...
+    super(props);
+    this.state = {
+      bgColor: {
+        start: Object.keys(BlueColor).indexOf('200'),
+        end: Object.keys(BlueColor).indexOf('0'),
+      },
+    };
+  }
+getFullName = claim => `${claim.firstName} ${claim.lastName}`
+listExpand = panelIndex => (event, expanded) => {
```
Comment (ender503) [inline components/UserList/index.js]: Please fix indents with `node_modules/eslint --fix components`

File: pages/user.js [hunk 3]
```diff
@@ -16,18 +16,79 @@ import fetchMamMessages from '../actions/fetchMamMessages';
...
+    // color: "#424242",
+    color: '#000000',
+    'font-size': '2.0em',
+    'font-weight': 'bolder',
+  },
+  messageInput: {
+    display: 'inline',
+    'margin-left': '1.2em',
+  },
+  Button_Coler: {
```
Comment (ender503) [inline pages/user.js]: `Button_Color` ? 🤔

File: pages/user.js [hunk 4]
```diff
@@ -16,18 +16,79 @@ import fetchMamMessages from '../actions/fetchMamMessages';
...
+    height: '45%',
+    width: '45%',
+  },
+  'qcode-word': {
+    // color: "#000000",
+    color: '#424242',
+    'font-weight': 'bolder',
+  },
+  'qcode-Name': {
+    // color: "#424242",
```
Comment (ender503) [inline pages/user.js]: Please remove unnecessary comments 👍

**Comment by jserv** [comment]
There are several functional commits in branch [test-ui-dev](https://github.com/jamestsay1207/TangleID/commits/test-ui-div), looking much better than squashed one you requested. Split this commit into smaller and dedicated ones.

**Comment by jserv** [comment]
Does this pull request depend on PR #44 ?

File: components/UserList/index.js [hunk 5]
```diff
@@ -1,21 +1,113 @@
...
+    flexGrow: 1,
+  },
+  chatroom: {
+    display: 'inline',
+    margin: '0 10px',
+  },
+  'list-expand': {
+    border: '1px solid #0288D1',
+  },
+  Word_Coler: {
```
Comment (jkrvivian) [inline components/UserList/index.js]: Word_Color?

**Comment by jserv** [comment]
@jamestsay1207, You have to improve the subject of Git commit messages including this pull request. Descriptions shall be informative.

**Comment by jserv** [comment]
Since we have changed the plans of presentation layer, UI-specific pull requests would be dropped.

**Pull request closed without merging**